src/Cmts.ml

```diff
@@ -490,10 +490,35 @@ let fmt_cmt (conf : Conf.t) ~fmt_code (cmt : Cmt.t) =
             | _, None -> str line $ fmt "*)"
             | _, Some _ -> str line $ fmt "@,*") )
   in
-  let fmt_non_code (cmt : Cmt.t) =
+  let fmt_unwrapped_cmt ({txt= s; loc} : Cmt.t) =
+    let lines = String.split_lines s in
+    let first_line_empty =
+      List.length lines > 0 && String.is_empty (List.hd_exn lines)
+    in
+    let begins_line = Source.begins_line t.source loc ~ignore_spaces:false in
+    if String.contains s '\n' && (not begins_line) && not first_line_empty
+    then
+      let b_space = String.length s > 0 && Char.equal s.[0] ' ' in
+      let e_space =
+        String.length s > 0 && Char.equal s.[String.length s - 1] ' '
```

Comment:
```suggestion
        String.length s > 1 && Char.equal s.[String.length s - 1] ' '
```
To avoid picking the same character as for `b_space` ?

Reply:
I agree, but this code (and more) could be simplified since we suppose `s` contains an `\n` anyway (not a fan of hidden assumptions though)

Reply:
It's not really an hidden assumption, the `if` is 2 lines above !
I agree this constant deserves a comment.